src/App.jsx

```diff
@@ -85,11 +100,14 @@ const App = () => {
       <SelectUserContext.Provider value={selectedUserObject}>
         <UserListContextProvider session={session}>
           <InboxMessageContext.Provider value={inboxMessageObject}>
-            <AppRoutes
-              isLoggedIn={session.info.isLoggedIn}
-              loadingActive={false}
-              loadMessages={loadMessages}
-            />
+            <OutboxMessageContext.Provider value={outboxMessageObject}>
```

Comment:
Is this inbox data universal to the app? It seems to me only components under inbox and outbox would be interested in it. If that's the case, this context would probably be better in that tab.

Reply:
Two duplicate comments.